src/test/java/com/learnwithted/kidbank/domain/PercentTest.java

```diff
@@ -0,0 +1,23 @@
+package com.learnwithted.kidbank.domain;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PercentTest {
+
+  @Test
+  public void testToString() {
+    Assert.assertEquals("100%",
+            Percent.of(1).over(0).toString());
+    Assert.assertEquals("50%",
+            Percent.of(1).over(2).toString());
+  }
+
+  @Test
+  public void testRoundToInteger() {
```

Comment:
You could add a test that returns `99` (the RHS of the ternary operator on line 29 of roundToInteger())

Reply:
Thank you I always miss these when they use the single line if statements